packages/web-react/src/components/Tooltip/README.md

```diff
@@ -220,4 +220,73 @@ const { getReferenceProps, getFloatingProps } = useInteractions([hover, focus, d
+| `id`                            | `string`                                     | -        | ✔        | Tooltip id                                                                                                                                                                                                                                                                 |
+| `isDismissible`                 | `bool`                                       | false    | ✕        | Make tooltip dismissible                                                                                                                                                                                                                                                   |
+| `isOpen`                        | `bool`                                       | -        | ✔        | Open state                                                                                                                                                                                                                                                                 |
+| `onToggle`                      | `() => void`                                 | -        | ✔        | Function for toggle open state of dropdown                                                                                                                                                                                                                                 |
+| `placement`                     | [Placement Dictionary][dictionary-placement] | "bottom" | ✕        | Placement of tooltip                                                                                                                                                                                                                                                       |
+
 [dictionary-placement]: https://github.com/lmc-eu/spirit-design-system/tree/main/docs/DICTIONARIES.md#placement
+[example]: https://spirit-design-system-demo.netlify.app/src/scss/components/tooltip/#advanced-positioning
+[floating-ui-flip-cross-axis]: https://floating-ui.com/docs/flip#crossaxis
+[floating-ui-flip-fallback-axis-side-direction]: https://floating-ui.com/docs/flip#fallbackaxissidedirection
+[floating-ui-flip-fallback-placements]: https://floating-ui.com/docs/flip#fallbackplacements
+[floating-ui-flip]: https://floating-ui.com/docs/flip
+[floating-ui-shift]: https://floating-ui.com/docs/shift
+[floating-ui-size]: https://floating-ui.com/docs/size
+[floating-ui]: https://floating-ui.com
+[readme-deprecations]: https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web/README.md#deprecations
```

Comment:
this should link to the web-react package
```suggestion
[readme-deprecations]: https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web-react/README.md#deprecations
```

Reply:
fixed